src/routes/key_value.py

```
@@ -20,12 +20,16 @@ async def get_kv_value(
 
 # This works as "api/kv" + "s" (="api/kvs")
 @kv_router.get("s")
-async def get_all_kv_values(
+async def get_kv_values(
     current_user: NullableUserDep,
     kv_service: KvServiceDep,
+    keys: Optional[list[str]] = Query(None, alias="q")
```

Ruff (0.15.9)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)